src/walk.rs

```diff
@@ -367,6 +367,8 @@ fn spawn_receiver(
                     handles.push(handle);
                 }
 
+                // TODO: once our MSRV supports scoped threads, it would probablly make sense to
+                // use that here
```

Comment:
Thank you for the reminder! Can we maybe remove the TODO and create an issue?

Reply:
Sure, I can do that.

Reply:
#1141